deploy/kubernetes/base/node.yaml

```diff
@@ -26,7 +25,6 @@ spec:
           securityContext:
             privileged: true
           image: amazon/aws-efs-csi-driver:latest
-          imagePullPolicy: Always
```

Comment:
Same above

Reply:
imagePullPolicy was not added in helm (so no need to change).